Added element type section (#4442)
umbraco/UmbracoDocs: Fundamentals/Data/Defining-content/index.md

## What is a Document Type?

A Document Type contains fieldsets (or groups) where you can apply rules about where the content can be created, allowed template(s), backoffice icons, etc.

Document Types can define entire pages or more limited content that can be reused on other nodes ie. a Search Engine Optimization (SEO) group. This means that you are in complete control of what type of content can be created where.

Document Types define what an end user sees and can interact with when they are working in the Umbraco backoffice. For a "Blog post" Document Type containing a thumbnail, a name, and an author image, it needs to be defined in the Document Type. All blog posts using the "Blog post" Document Type, will allow the end user to fill in a thumbnail, author name, and an author image.

### Properties

> Each field on a Document Type is called a property. A property is given a name, an alias (used to output the properties contained in a template), and an editor. The editor determines what type of data the property will store and the input method. There is a wide range of property editors available out-of-the box (Textstring, Richtext, Media Picker, and so forth) and you can customize additional editors.

Reviewer: ```suggestion
Each field on a Document Type is called a property. The property is given a name, an alias (used to output the properties contained in a template), and an editor. The editor determines what type of data the property will store and the input method. There is a wide range of default property editors available (Textstring, Richtext, Media Picker, and so forth) and you can customize additional editors.
```

### Creating child nodes

Before creating a Text Page in __Content__ section, allow the Text Page Document Type to be created as a child node to the Home node. Select the Home Document Type and go to the __Permissions__ group. Click __Add child__ and select Text Page.

![Allowing child nodes](images/v8Screenshots/setPagePermissions.png)

Go to the __Content__ section and click the menu icon (•••) next to the *Home* node and select the Text page Document Type. We'll name the page "About us". We now have a basic content structure.

![Basic content structure](images/v8Screenshots/createAboutUs.png)

Document Types are flexible and can be used for defining pieces of reusable content or an entire page, to acting as a container or repository.

## What is an Element Type?

> An Element Type is a Document Type without a template containing smaller schema configurations for repeating a set of properties. These can be used for defining schema in the Block List Editor, Nested Content, Block Grid Editor, or other Element Type based editors. Element Types cannot be used to create content that resides in the Content tree.

Reviewer: ```suggestion
An Element Type is a Document Type without a template containing schema configurations for repeating a set of properties. These are for defining schema in the Block List Editor, Nested Content, Block Grid Editor, or other Element Type based editors. Element Types cannot be used to create content that resides in the Content tree.
```

> When you create an Element type, it automatically sets the **Is Element Typw** flag to **True** on the **Permissions** tab.

Reviewer: ```suggestion
When you create an Element type, it automatically sets the **Is Element Type** flag to **True** on the **Permissions** tab.
```

![Element Type](images/Element-Type.png)

## Creating an Element Type

An Element Type is created under Document Types in the __Settings__ section.

Go to the __Settings__ section in the backoffice. On the __Document Types__ node click the menu icon (•••) to bring up the context menu. Here, choose __Element Type__. This will create an Element Type.

> ![Creating an Element Type](images/Element-Type-create.png)

Reviewer: I don't think it's necessary to add steps on how to create an Element Type.
The entire article is about how to create a Document Type, which is pretty much the same workflow.

Perhaps instead, something like this could be added:
"Element Types are created using the same workflow as regular Document Types, but usually contain fewer properties. You can also create Element Types as part of configuring a Block Grid or Block List Data Type."

What do you think? 🤔

Author: Yup, makes sense. I'll remove the section and add the suggested line instead 💪
